packages/blocks/src/api/factory.js

```diff
@@ -457,6 +454,24 @@ export function getBlockTransforms( direction, blockTypeOrName ) {
 	} ) );
 }
 
+/**
+ * Checks that a given transforms isMatch method passes for given source blocks.
+ *
+ * @param {Object}       transform   A transform object.
+ * @param {Array|Object} blocks      Blocks array.
+ * @param {Object}       sourceBlock Source block being transformed.
+ *
+ * @return {boolean} True if given blocks are a match for given tranform.
+ */
+function checkTransformIsMatch( transform, blocks, sourceBlock ) {
```

Comment:
Seems like blocks or sourceBlock is the same thing right?, can't we just use a single argument for this?

Reply:
Exactly source block is just the first block in blocks `const sourceBlock = first( blocks );`, we can have a single parameter.

Reply:
thanks @youknowriad & @jorgefilipecosta, good point about the unnecessary argument, I have tidied this up.